loutilities/flask_helpers/mailer.py

```diff
@@ -7,11 +7,12 @@
 # pypi
 from flask import current_app
 from flask_mail import Message
+import mimetypes
```

Comment:
My style has been the following

`from mimetypes import guess_type`

Reply:
Updated